### DISCUSSION TITLE
Ricedroidoss clean sync please

### DISCUSSION THREAD

**Discussion by ghost** [ROM-builders/temporary#17312] [General]

**Comment by Apon77** [comment]
why!

**Reply by ghost** [reply]
> why!

![Screenshot_20230126-192302_Chrome](https://user-images.githubusercontent.com/114152118/214846923-a8e86878-be44-4578-b4b1-f799f569538a.png)

**Comment by Apon77** [comment]
Link?

**Reply by ghost** [reply]
https://cirrus-ci.com/github/ROM-builders/temporary/pull%2F17246

**Reply by ghost** [reply]
A clean sync really can help. Since new commit pushed for sync fixing. Many of them tired of getting ricedroidoss sync error.

**Reply by ghost** [reply]
@Apon77 

https://cirrus-ci.com/task/4706092907757568

Failed again

Clean sync please

**Reply by ghost** [reply]
I beg you sir.. Just once clean sync for me 🥲🥲

**Comment by ghost** [comment]
https://cirrus-ci.com/task/5747295656869888
Failing
